packages/clerk-js/src/ui/foundations/colors.ts

```diff
@@ -66,7 +65,21 @@ export const colors = Object.freeze({
   primary700: '#25232A',
   primary800: '#201D23',
   primary900: '#1B171C',
-  ...colorOptionToHslaAlphaScale('#2F3037', 'primaryAlpha'),
+  ...colorOptionToHslaAlphaScale('#2F3037', 'primaryAlpha')!,
+  primaryForeground: '#FFFFFF',
+  secondary25: '#FFFFFF',
+  secondary50: '#FFFFFF',
+  secondary100: '#FFFFFF',
+  secondary200: '#FFFFFF',
+  secondary300: '#FFFFFF',
+  secondary400: '#FFFFFF',
+  secondary500: '#FFFFFF',
+  secondary525: '#F7F7F8',
```

Comment:
@desiprisg we should remove the 525 right?

Reply:
removing..